packages/core/cms/sections.json

```diff
@@ -46,30 +326,253 @@
       }
     }
   },
+  {
+    "name": "Footer",
+    "schema": {
+      "title": "Footer",
+      "description": "Footer displayed on all pages",
+      "type": "object",
+      "required": [],
+      "properties": {
+        "incentives": {
+          "title": "Incentives",
+          "type": "array",
+          "minItems": 3,
+          "maxItems": 5,
+          "items": {
+            "title": "Incentive",
+            "type": "object",
+            "required": ["title", "firstLineText", "icon"],
+            "properties": {
+              "title": {
+                "type": "string",
+                "title": "Title"
+              },
+              "firstLineText": {
+                "type": "string",
+                "title": "First line of text"
+              },
+              "secondLineText": {
+                "type": "string",
+                "title": "Second line of text"
+              },
+              "icon": {
+                "type": "string",
+                "title": "Icon",
+                "enumNames": [
+                  "Truck",
+                  "Calendar",
+                  "Gift",
+                  "Store Front",
+                  "Shield Check"
+                ],
+                "enum": [
+                  "Truck",
+                  "Calendar",
+                  "Gift",
+                  "Storefront",
+                  "ShieldCheck"
+                ]
+              },
+              "alt": {
+                "title": "Alternative Label",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "footer-links": {
```

Comment:
Should this be required?

Reply:
Not necessarily, without links, the layout doesn't broke